apps/web/src/server/applications/case/documentation/applications-documentation.controller.js

```diff
@@ -252,6 +252,10 @@ export async function viewApplicationsCaseDocumentationProperties({ session }, r
 	const documentationFile = await getCaseDocumentationFileInfo(caseId, documentGuid);
 	const documentVersions = await getCaseDocumentationFileVersions(documentGuid);
 
+	if (documentationFile.stage === 'Correspondence') {
+		documentationFile.stage = '';
```

Comment:
rather than clearing it in the display, should we rather be setting it to null in the folders template?

Reply:
happy to have a look with you when we get a moment!